packages/client-sdk-nodejs/src/internal/leaderboard-data-client.ts

```diff
@@ -58,33 +60,44 @@ export class LeaderboardDataClient implements ILeaderboardDataClient {
       `Creating leaderboard client using endpoint: '${this.credentialProvider.getCacheEndpoint()}'`
     );
 
-    this.clientWrapper = new IdleGrpcClientWrapper({
```

Comment:
this part is mostly a whitespace change; check github's "hide whitespace" to make this diff more obvious